backend/dataall/modules/datasets/api/dataset/resolvers.py

```diff
@@ -201,13 +204,21 @@ def validate_creation_request(data):
             raise RequiredParameter('group')
         if not data.get('label'):
             raise RequiredParameter('label')
+        RequestValidator.validate_confidentiality(data)
         if len(data['label']) > 52:
             raise InvalidInput(
                 'Dataset name', data['label'], 'less than 52 characters'
             )
 
     @staticmethod
-    def validate_import_request(data):
+    def validate_confidentiality(data):
+        confidentiality = data.get('confidentiality') if not custom_confidentiality_mapping else custom_confidentiality_mapping[data.get('confidentiality')]
```

Comment:
can we use the helper method or create a new helper method in `class ConfidentialityClassification()` so we do not need to also import config here as
`custom_confidentiality_mapping = config.get_property('modules.datasets.features.custom_confidentiality_mapping', {})
`

Similar to what you do in with `backend/dataall/modules/datasets/services/dataset_column_service.py`:
```
ConfidentialityClassification.get_confidentiality_level(dataset.confidentiality, context) != ConfidentialityClassification.Unclassified.value```

Reply:
Okay. I see the value of it. Thanks for the suggestion. I will refactor it!